CONTRIBUTING.md

```diff
@@ -57,15 +60,15 @@ entire environment configuration process.
 Otherwise, follow the steps below to manually configure your
 environment.
 
-#### Installing and using `pyenv` and `pyenv-virtualenv` ####
+#### Installing and using `gnu-getopt`, `pyenv`, and `pyenv-virtualenv` ####
 
 On the Mac, we recommend installing [brew](https://brew.sh/).  Then
-installation is as simple as `brew install pyenv pyenv-virtualenv` and
+installation is as simple as `brew install gnu-getopt pyenv pyenv-virtualenv` and
 adding this to your profile:
 
 ```bash
 export PYENV_ROOT="$HOME/.pyenv"
-export PATH="$PYENV_ROOT/bin:$PATH"
+export PATH="$PYENV_ROOT/bin:$(brew --prefix)/opt/gnu-getopt/bin:$PATH"
```

Comment:
On my system, if I don't add that blurb (`$(brew --prefix)/opt/gnu-getopt/bin`) to my `PATH`, it uses the version of `getopt` provided by macOS:

```
# Default state
❱ which getopt
/usr/bin/getopt

# After PATH update
❱ which getopt
/opt/homebrew/opt/gnu-getopt/bin/getopt
```

Reply:
Agh you are right. It's keg-only on macOS and I have a file in my `.bashrc.d/` that adds it to the path with other keg-only formulae that I want to use. Fixing the PATH for `getopt` should be separate from `pyenv` environment configuration so this suggestion is still something I want, but it should be moved instead of completely deleted.

Reply:
> Agh you are right. It's keg-only on macOS and I have a file in my `.bashrc.d/` that adds it to the path with other keg-only formulae that I want to use. Fixing the PATH for `getopt` should be separate from `pyenv` environment configuration so this suggestion is still something I want, but it should be moved instead of completely deleted.

In full disclosure, @michaelsaki originally had the two `PATH` adjustments separate and I asked him to munge them.  _MY IGNORANCE ASTOUNDS AND AMAZES!!!_

Reply:
@jsf9k Obviously I was accounting for @mcdonnnj logic when I made the two `PATH`s 👀

Reply:
@mcdonnnj I addressed this here [495862a](https://github.com/cisagov/skeleton-generic/pull/166/commits/495862aeaf3430b0a69f92cf54b6415818139db2)